3_visualize/src/plot_timeseries.R

```diff
@@ -1,6 +1,6 @@
-plot_nwis_timeseries <- function(fileout, site_data_styled, width = 12, height = 7, units = 'in'){
-  
-  ggplot(data = site_data_styled, aes(x = dateTime, y = water_temperature, color = station_name)) +
+plot_nwis_timeseries <- function(fileout, site_dat, width = 12, height = 7, units = 'in'){
```

Comment:
FWIW, I personally prefer a pattern of naming filename arguments so that it's obvious they're file names as opposed to objects, e.g., `site_data_file`. But that convention wasn't being followed in this repo even before this PR, so doesn't seem worth pushing here.